service/server.py

```diff
@@ -175,24 +178,42 @@ def find_titles():
                 session['title'] = title
                 # Redirect to the display_title method to display the digital
                 # register
-                return redirect(
-                    url_for('display_title', title_ref=search_term.upper()))
-        # If search value doesn't match, return no results found screen
-        return render_template(
-            'no_title_number_results.html',
-            asset_path='../static/',
-            search_term=search_term,
-            google_api_key=GOOGLE_ANALYTICS_API_KEY,
-            form=TitleSearchForm()
-        )
-    else:
-        # If not search value enter or a GET request, display the search page
-        return render_template(
-            'search.html',
-            asset_path='../static/',
-            google_api_key=GOOGLE_ANALYTICS_API_KEY,
-            form=TitleSearchForm()
-        )
+                return redirect(url_for('display_title', title_ref=search_term.upper()))
+            else:
+                # If title not found display 'no title found' screen
+                return render_template('search_results.html',
```

Comment:
Now that PEP8 adjustments have made this method very long, we could think about extracting the `render_template` calls into separate functions, such as `render_search_results`.